ES/ESmy023.xml

```diff
@@ -32,99 +32,377 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-023</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Gadla Yoḥannǝs Maṭmǝq “Vita of John the Baptist” (fols. 4ra-78ra)&#xD;
+                  <summary>I) Gadla Yoḥannǝs Maṭmǝq “Vita of John the Baptist” (fols. 4ra-78ra);
                      II) Taʾammǝra Yoḥannǝs Maṭmǝq “Miracles of John the Baptist”: 13 miracles (fols.
-                     78ra-88vb)&#xD; III) Malkǝʾa Yoḥannǝs Maṭmǝq “Image of John the Baptist”: 28
-                     strophes (fols. 89rb-90va)&#xD; &#xD; Colophon.</summary>
-                  <msItem xml:id="ms_i1">
+                     78ra-88vb); III) Malkǝʾa Yoḥannǝs Maṭmǝq “Image of John the Baptist”: 28
+                     strophes (fols. 89rb-90va); Colophon.</summary>
 
-                     <incipit>
-                        <foreign xml:lang="en"/>
+                    <msItem xml:id="ms_i1">
```

Comment:
I repeat: I would introduce an overarching msItem around for the collection (dossier) dedicated to JB, it does not matter if it is not a fixed collection.
See the examples for other hagiographic mss which I already gave.
You need an overarching msItem anyway, even without a ref, in order to accommodate the colophon, which belongs to the whole collection.

Reply:
I responded to this.